### PR TITLE
make handControllerGrab.js work with new API

### PR DESCRIPTION
**Pull request by ZappoMan**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 3e7364608bbc7e27283d93767e996d515b975832 available at: https://gist.github.com/58954a809c0841d2f400<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6149.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6149.dmg

File: examples/controllers/handControllerGrab.js [hunk 1]
```diff
@@ -856,8 +858,8 @@ function MyController(hand, triggerAction) {
     };
 }
 
-var rightController = new MyController(RIGHT_HAND, Controller.findAction("RIGHT_HAND_CLICK"));
-var leftController = new MyController(LEFT_HAND, Controller.findAction("LEFT_HAND_CLICK"));
+var rightController = new MyController(RIGHT_HAND, Controller.Standard.RT);
```
Comment (jherico) [inline examples/controllers/handControllerGrab.js]: incorrect, should be `Controller.Actions.RightHandClick`

File: examples/controllers/handControllerGrab.js [hunk 2]
```diff
@@ -856,8 +858,8 @@ function MyController(hand, triggerAction) {
...
+var leftController = new MyController(LEFT_HAND, Controller.Standard.LT);
```
Comment (jherico) [inline examples/controllers/handControllerGrab.js]: same

File: interface/resources/controllers/standard.json [hunk 3]
```diff
@@ -12,17 +12,8 @@
...
-            "to": "Actions.BOOM_IN", 
-            "filters": [ { "type": "scale", "scale": 0.1 } ]
-        },
-
-        { 
-            "from": "Standard.LT", 
-            "to": "Actions.BOOM_OUT", 
-            "filters": [ { "type": "scale", "scale": 0.1 } ]
-        },
+        { "from": "Standard.LT", "to": "Actions.LEFT_HAND_CLICK" },
```
Comment (jherico) [inline interface/resources/controllers/standard.json]: should use the CamelCase version, Actions.LeftHandClick

File: examples/controllers/handControllerGrab.js (hunk 1 above)
Comment (ZappoMan) [inline examples/controllers/handControllerGrab.js]: this is the discussion we were having... in slack, what's the difference between wiring to the Standard input vs the action... both work

**Comment by hifi-gustavo** [comment]
Build console output for commit: 67f2a0c6c803263171a98ab02588079beb1a4ba8 available at: https://gist.github.com/58954a809c0841d2f400<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6149.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6149.dmg

**Pull request merged**